src/unix/core.c

```diff
@@ -35,7 +35,10 @@
 #include <sys/un.h>
 #include <netinet/in.h>
 #include <arpa/inet.h>
-#include <limits.h> /* INT_MAX, PATH_MAX */
+#ifndef __need_IOV_MAX
+#define __need_IOV_MAX
+#endif
```

Comment:
This is to pry IOV_MAX out of glibc, right?  I'm not sure if that's a good idea because it used to be lower than the current limit of 1024.  IIRC, it was something like 64.

That might become a problem if someone compiles libuv against headers that are newer than the kernel it runs on.  I think that 2.6.9 (the oldest kernel we support) is unaffected by that but I'd rather err on the side of caution.